registry/remote/auth/credential.go

```diff
@@ -38,3 +40,14 @@ type Credential struct {
 	// Reference: https://docs.docker.com/registry/spec/auth/token/
 	AccessToken string
 }
+
+// StaticCredential is a utility method for specifying static credentials that
+// are used when creating a client.
+func StaticCredential(registry string, cred Credential) func(ctx context.Context, reg string) (Credential, error) {
```

Comment:
Nit: rename `registry` to `host` and `reg` to `target`

Reply:
nice catch!

Reply:
changed accordingly.